<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBClientInfoPage</class>
 <widget class="QFrame" name="OpenRGBClientInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Client Info Page</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="6">
    <widget class="QLineEdit" name="ClientPortValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="ClientPortLabel">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="ClientConnectButton">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="ClientIPValue"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="ClientIPLabel">
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="6">
    <widget class="QTreeWidget" name="ClientTree">
     <property name="columnCount">
      <number>5</number>
     </property>
     <column>
      <property name="text">
       <string>Connected Clients</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Protocol Version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Save Connection</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ClientIPValue</tabstop>
  <tabstop>ClientPortValue</tabstop>
  <tabstop>ClientConnectButton</tabstop>
  <tabstop>ClientTree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
